include/imex/Dialect/PTensor/IR/PTensorOps.h

```diff
@@ -53,6 +53,7 @@ enum EWBinOpId : int {
   NOT_EQUAL,
   OR,
   POWER,
+  RSHIFT,
```

Comment:
I guess LSHIFT and RSHIFT are duplicates of BITWISE_LEFT_SHIFT and BITWISE_RIGHT_SHIFT.

Reply:
@fschlimb Okay, we can just duplicate them.

Reply:
Well, I actually thought we should not. Any reason for doing so?

Reply:
Well, actually there was `LSHIFT` but no 'RSHIFT' so I added it.

Reply:
I prefer BITWISE_*_SHIFT.